<launch>
  <!-- Folder of images to be labeled -->
  <arg name='source_path' default='$(find nist_atb_eval)/data/test' />
  <arg name='starting_position' default='0' />
  <!-- Output CSV for assigned labels. NOTE: If the file already exists, lines will be appended -->
  <arg name='save_path' default='$(arg source_path)' />
  <arg name='save_file' default='tb_labels.csv' />
  <!-- CSV file defining taskboard ROIs -->
  <arg name='roi_csv' default='$(find nist_atb_eval)/config/tb_roi.csv' />

  <node name='taskboard_data_labeler' pkg='nist_atb_eval' type='data_labeler.py' output='screen' required='true'>
    <param name='source_path' value='$(arg source_path)' />
    <param name='starting_position' value='$(arg starting_position)' />
    <param name='save_path' value='$(arg save_path)' />
    <param name='save_file' value='$(arg save_file)' />
    <param name='roi_csv' value='$(find nist_atb_eval)/config/tb_roi.csv' />
  </node>
</launch>
